.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -32,6 +32,11 @@ jobs:
   timeoutInMinutes: 360
 
   steps:
+  - script: |
+        rm -rf /opt/ghc
+        df -h
+    displayName: Manage disk space (rm /opt/ghc)
```

Comment:
This works too.

Reply:
Thanks for checking 😄

Reply:
Submitted upstream in PR ( https://github.com/conda-forge/conda-smithy/pull/1345 ).

Reply:
Thanks @jakirkham. I'm thinking we can probably close this one, and just rerender #8 after the smithy-PR lands.